src/mo_frontend/static.ml

```diff
@@ -99,7 +99,7 @@ and exp_fields m efs = List.iter (fun (ef : exp_field) ->
 and dec m d = match d.it with
   | TypD _ | ClassD _ -> ()
   | ExpD e -> exp m e
-  | LetD (p, e) -> pat m p; exp m e
+  | LetD (p, e, fail) -> pat m p; exp m e; Option.iter (exp m) fail
```

Comment:
Note to self: I guess this will error pretty soon after entering any fail clause since all control flow construct error.